rocky/katalogus/client.py

```diff
@@ -144,29 +164,56 @@ def get_cover(self, boefje_id: str) -> BytesIO:
         return BytesIO(response.content)
 
 
-def parse_plugin(plugin: Dict) -> Plugin:
-    try:
-        consumes = {type_by_name(consumes) for consumes in plugin["consumes"]}
-    except StopIteration:
-        consumes = set()
+def parse_boefje(boefje: Dict) -> Boefje:
+    scan_level = SCAN_LEVEL(boefje["scan_level"])
 
+    consumes = set()
     produces = set()
-    for ooi in plugin["produces"]:
-        with contextlib.suppress(StopIteration):
-            produces.add(type_by_name(ooi))
-
-    return Plugin(
-        id=plugin["id"],
-        type=plugin["type"],
-        name=plugin.get("name") or plugin["id"],
-        repository_id=plugin["repository_id"],
-        description=plugin["description"],
-        scan_level=SCAN_LEVEL(plugin["scan_level"]),
-        consumes=consumes,  # TODO: check if we still want to support multiple
+    with contextlib.suppress(StopIteration):
+        consumes.add(type_by_name(boefje["consumes"]))
+        produces.add(type_by_name(boefje["produces"]))
+
+    return Boefje(
+        id=boefje["id"],
+        repository_id=boefje["repository_id"],
+        name=boefje.get("name") or boefje["id"],
+        description=boefje["description"],
+        enabled=boefje["enabled"],
+        type=boefje["type"],
+        scan_level=scan_level,
+        consumes=consumes,
         produces=produces,
-        enabled=plugin["enabled"],
     )
 
 
+def parse_normalizer(normalizer: Dict) -> Normalizer:
+    # TODO: give normalizers a proper name in backend
+    name = normalizer["id"].replace("_", " ").replace("kat ", "").title()
```

Comment:
We use the id if there is no name for boefjes (`name=boefje.get("name") or boefje["id"]`), so we could do that here as well?

Reply:
For now having underscores all over in the KAT-alogus does not look great. This is a temporary fix till we fill the name, if this really fails it falls back to the id.

Reply:
Shouldn't we update the boefjes as well then?